packages/itwinui-css/src/divider/divider.scss

```diff
@@ -0,0 +1,25 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+
+@mixin iui-divider {
```

Comment:
all divider styles should go under the `utils` folder, next to `flex`

Reply:
Moved to `utils` folder.